tools/testrunner/base_runner.py

```diff
@@ -105,17 +105,8 @@
 
 # Increase the timeout for these:
 SLOW_ARCHS = [
-  "arm",
-  "arm64",
-  "mips",
-  "mipsel",
-  "mips64",
-  "mips64el",
-  "s390",
-  "s390x",
-  "riscv64",
-  "riscv32",
-  "loong64"
```

Comment:
don't change this style

Reply:
I have changed the patch， rebase on the previous format code.